app/Comment.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Comment extends Model
+{	
+	protected $guarded = [
+		'id',
+	];
+
+    public function ticket()
+	{
+	    return $this->belongsTo('App\Ticket');
```

Comment:
```suggestion
	    return $this->belongsTo(Ticket::class);
```
chuyển sang và use Model vào nhé